app/javascript/controllers/application.js

```diff
@@ -1,6 +1,8 @@
 import { Application } from "@hotwired/stimulus"
+import TypedController from "../controllers/typed_controller"
 
 const application = Application.start()
+application.register("typed", TypedController)
```

Comment:
This too

Reply:
this one I need, it's not working without it.